source/includes/items/_header.md

```diff
@@ -0,0 +1 @@
+# Items (assets)
```

Comment:
should everything be listed as assets or items/both?

Reply:
Good question, probably need to discuss the correct terminology we should use throughout - we only use "items" because "assets" is reserved word